content/help/faq/security/istio-to-not-istio.md

```diff
@@ -1,5 +1,8 @@
 ---
-title: Can a service with Istio Auth enabled communicate with a service without Istio?
+title: Can a service with Istio mutual TLS (mTLS) enabled talk to a service without Istio,
```

Comment:
Can we have a shorter title? Super-long titles don't look too good on the main FAQ page.

Reply:
Done.